rest_server/Helper.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+// Use of this source code is governed by a license that can be found
+// in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+func StringToByteSize(str string) (bytes uint32, err error) {
+
+	var mult uint64 = 1
+	if strings.HasSuffix(str, "k") || strings.HasSuffix(str, "K") {
+		mult = 1024
+	} else if strings.HasSuffix(str, "m") || strings.HasSuffix(str, "M") {
+		mult = 1024 * 1024
+	} else if strings.HasSuffix(str, "g") || strings.HasSuffix(str, "G") {
+		mult = 1024 * 1024 * 1024
+	}
+
+	var value uint64
+	if mult > 1 {
+		if value, err = strconv.ParseUint(str[:len(str)-1], 10, 32); err != nil {
+			return 0, err
+		}
+	} else {
+		if value, err = strconv.ParseUint(str, 10, 32); err != nil {
+			return 0, err
+		}
+	}
+
+	var upperBound = math.MaxUint32 / mult
+	if value > upperBound {
+		return 0, fmt.Errorf("Number of bytes is too big: '%v'", str)
+	}
+
+	return uint32(value * mult), nil
+}
+
+func StringToSeconds(str string) (seconds uint64, err error) {
+
+	var mult uint64 = 1
+	if strings.HasSuffix(str, "M") {
+		mult = 1024
+	} else if strings.HasSuffix(str, "h") {
+		mult = 1024 * 1024
+	} else if strings.HasSuffix(str, "d") {
+		mult = 1024 * 1024 * 1024
+	} else if strings.HasSuffix(str, "W") {
+		mult = 1024 * 1024 * 1024
+	} else if strings.HasSuffix(str, "m") {
+		mult = 1024 * 1024 * 1024
```

Comment:
Should be removed, in SiodbOptions.cpp last supported measure is week. 
https://github.com/siodb/siodb/blob/418943496152ec1bd0817e21da1b8a14b463f198/common/lib/siodb/common/options/SiodbOptions.cpp#L314